Services/CO.CDP.EntityVerification/Services/PponService.cs

```diff
@@ -0,0 +1,11 @@
+namespace CO.CDP.EntityVerification.Services;
+
+public class PponService : IPponService
+{
+    public string GeneratePponId(string scheme, string departmentIdentifier)
+    {
+        var id = Guid.NewGuid().ToString().Replace("-", string.Empty);
+
+        return $"{scheme}-{id}-{departmentIdentifier}";
```

Comment:
The scheme should be removed, as we store identifiers separate to the scheme. The way it's returned here, looks like a formatted PPON. We should return the identifier only and persist that.

Reply:
Understand, done.